derived/processors.adoc

```diff
@@ -0,0 +1,133 @@
+:serialization: ../serialization/serialization
+:m3: ../metametamodel/metametamodel
+:bulk: ../bulk/repo-access-api
+:chunk: <<{serialization}.adoc#SerializationChunk, SerializationChunk>>
+
+= Processors and Derived Models
+:toc: preamble
+:toclevels: 2
+
+== Terminology
+
+repository::
+A _repository_ is the "world".
+- A repository is the scope in which id's are unique.
+- A repository is the scope in which referred nodes can be resolved.
+- A repository is a collection of partitions.
+
+original node::
+A node CRUDed by users (mediated by tools).
+
+derived node::
+A node implementing `IDerived` that is attached (by reference) to one base node and
+that can be calculated from its base node and (optionally) other nodes.
+
+calculated node:: A node that is (automatically) calculated from other nodes.
+A derived node is a calculated node, but not vice versa.
+
+model::
+A _model_ is a subset of a repository.
+- A model is a collection of partitions (from the same repository).
+
+original model::
+An _original model_ is a model where all nodes are original nodes.
+- An original model cannot be (re-)computed from other models.
+
+derived model::
+A _derived model_ is a model that can be calculated from _base_ models.
+- A derived model has no original nodes.
+- In a derived model all nodes are created (automatically) by a processor. 
+  It contains derived nodes and calculated nodes.
+
++
+However, a derived model MAY contain nodes that do not implement `IDerived`.
+Example: A derived richtext description contains `Word` concepts.
+`Word` is reused in many contexts, and does not implement `IDerived`.
+In this case `Word` is a calculated node, but *not* a derived node.
+
+NOTE: A repository always contains exactly one original model and any number of derived models.
```

Comment:
Only one original model? Not any number of models?

Reply:
It contains any number of original partitions.